src/main/webapp/css/custom-prism.css

```diff
@@ -50,6 +50,11 @@ pre > code.highlight {
     transition: background-color .2s ease;
 }
 
+.analysis-detail > p {
+    color: black;
+    background-color: white;
+}
+
```

Comment:
Wouldn't it make more sense to use our CSS variables for all colors here?

Example: https://github.com/jenkinsci/bootstrap5-api-plugin/blob/d7465996904cddad89c8f57897159edc06d58201/src/main/webapp/css/jenkins-style.css#L25

Reply:
Good shout, that looked a little weird in dark mode against the p tag, so I've shifted it up to the class instead
<img width="1173" alt="image" src="https://github.com/user-attachments/assets/98f0f4a3-acc8-4aab-a2fc-ce42cd921191" />
<img width="1173" alt="image" src="https://github.com/user-attachments/assets/098bb395-53f3-42f5-a37e-686a7b12979f" />